### ISSUE TITLE
[Bug]: Category reorder (drag and drop) does not work when on desktop mode on mobile

### ISSUE THREAD

**Issue by joel-jeremy** [actualbudget/actual#1194]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Drag and dropping categories to reorder them does not seem to work when using Chrome on desktop mode in mobile.

### Where are you hosting Actual?

Fly.io

### What browsers are you seeing the problem on?

Chrome

### Operating System

Mobile Device

**Comment by Kidglove57** [comment]
I am using Safari on mobile. I have it set to 85% zoom. In landscape mode I can reorder categories by drag and drop.

I can also do so using Chrome on iOS.

I therefore wonder if this problem is specific to Android?

**Comment by j-f1** [comment]
Can you try on https://actualbudget.netlify.app/? We recently upgraded our drag and drop library, which may have fixed the issue.

**Comment by joel-jeremy** [comment]
I am also encountering the same issue on https://actualbudget.netlify.app/

**Comment by Kidglove57** [comment]
Confirmed drop and drag working for me on iPhone using https://actualbudget.netlify.app/, but of course only in landscape mode.

**Comment by Kidglove57** [comment]
Yes I should have clarified. 23.6.0 but with the latest features included, that have been added since that version (tagged “edge” rather than ”latest” ).

It will be good to hear from any other Android users.

**Comment by j-f1** [comment]
It looks like we may need to use the touch drag-and-drop backend instead of the HTML5 backend on Android. See https://github.com/react-dnd/react-dnd/issues/1660 and https://react-dnd.github.io/react-dnd/docs/backends/touch. I’m not sure what the best way to decide which backend to is — maybe there’s some sort of feature test that could be run? @joel-jeremy since you have a device where it doesn’t work, if you’re willing to figure out a good test for which backend to use that would be much appreciated. (no pressure though!)

**Comment by joel-jeremy** [comment]
I could take a stab at it but I'm encountering a lot of linter issues right now with the latest master branch.

**Comment by j-f1** [comment]
You should be able to ignore the linter errors. If they don’t go away after running `yarn` again and including the commit in https://github.com/actualbudget/actual/pull/1199, please ask in discord or open an issue so we can debug further.

**Comment by joel-jeremy** [comment]
I am encountering this error:
![image](https://github.com/actualbudget/actual/assets/20313680/eb9b94f2-ca7e-413c-bf58-51647a18188b)

**Comment by MatissJanis** [comment]
Yes. Thanks for the ping.
